source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.ApplyDBMigrationsApp/Scripts/Model/202201200800 Create tables.sql

```diff
@@ -0,0 +1,334 @@
+    [GridAreaId] [nvarchar](50) NOT NULL,
+    [EffectiveDate] [datetime2](7) NOT NULL,
+    [ConnectionState] [int] NOT NULL,
+    [SettlementMethod] [int] NULL,
+    CONSTRAINT [PK_MeteringPoint] PRIMARY KEY NONCLUSTERED
+(
+[Id] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY],
+    CONSTRAINT [UC_MeteringPointId] UNIQUE NONCLUSTERED
+(
+[MeteringPointId] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+    ) ON [PRIMARY]
+    GO
+
+CREATE TABLE [dbo].[SchemaVersions](
+    [Id] [int] IDENTITY(1,1) NOT NULL,
+    [ScriptName] [nvarchar](255) NOT NULL,
+    [Applied] [datetime] NOT NULL,
+    CONSTRAINT [PK_SchemaVersions_Id] PRIMARY KEY CLUSTERED
+(
+[Id] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+    ) ON [PRIMARY]
+    GO
+
+CREATE TABLE [MessageHub].[AvailableChargeData](
+    [Id] [uniqueidentifier] NOT NULL,
+    [RecipientId] [nvarchar](35) NOT NULL,
+    [RecipientRole] [int] NOT NULL,
+    [BusinessReasonCode] [int] NOT NULL,
+    [ChargeId] [nvarchar](35) NOT NULL,
+    [ChargeOwner] [nvarchar](35) NOT NULL,
+    [ChargeType] [int] NOT NULL,
+    [ChargeName] [nvarchar](132) NOT NULL,
+    [ChargeDescription] [nvarchar](2048) NOT NULL,
+    [StartDateTime] [datetime2](7) NOT NULL,
+    [EndDateTime] [datetime2](7) NOT NULL,
+    [TaxIndicator] [bit] NOT NULL,
+    [TransparentInvoicing] [bit] NOT NULL,
+    [VatClassification] [int] NOT NULL,
+    [Resolution] [int] NOT NULL,
+    [RequestDateTime] [datetime2](7) NOT NULL,
+    [AvailableDataReferenceId] [uniqueidentifier] NOT NULL,
+    [SenderId] [nvarchar](35) NOT NULL,
+    [SenderRole] [int] NOT NULL,
+    CONSTRAINT [PK_AvailableChargeData] PRIMARY KEY NONCLUSTERED
+(
+[Id] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+    ) ON [PRIMARY]
+    GO
+
+CREATE TABLE [MessageHub].[AvailableChargeDataPoints](
+    [Id] [uniqueidentifier] NOT NULL,
+    [AvailableChargeDataId] [uniqueidentifier] NOT NULL,
+    [Position] [int] NOT NULL,
+    [Price] [decimal](14, 6) NOT NULL,
+    CONSTRAINT [PK_AvailableChargeDataPoints] PRIMARY KEY NONCLUSTERED
+(
+[Id] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+    ) ON [PRIMARY]
+    GO
+
+CREATE TABLE [MessageHub].[AvailableChargeLinksData](
+    [Id] [uniqueidentifier] NOT NULL,
+    [RecipientId] [varchar](70) NOT NULL,
+    [RecipientRole] [int] NOT NULL,
+    [BusinessReasonCode] [int] NOT NULL,
+    [ChargeId] [varchar](70) NOT NULL,
+    [ChargeOwner] [varchar](70) NOT NULL,
+    [ChargeType] [int] NOT NULL,
+    [MeteringPointId] [varchar](70) NOT NULL,
```

Comment:
Suggested to align with metering point table: 
```suggestion
    [MeteringPointId] [nvarchar](50) NOT NULL,
```